### ISSUE TITLE
DWR Colors

### ISSUE THREAD

**Issue by stdavis** [agrc/electrofishing-query#225]
not a blocker, but we may want to swap in dwr specific colors here eventually

_Originally posted by @steveoh in https://github.com/agrc/electrofishing-query/pull/223#discussion_r1811175160_
            
We may want to update the footer as well.

**Comment by stdavis** [comment]
The colors seem good now. I'm interested in your thoughts on the footer. I think that it's weird and perhaps off-putting to our client to have all of our stuff in the footer. Do we need to make it customizable? Or different versions for different departments?

**Comment by steveoh** [comment]
Yeah we rush jobbed our footer and haven't made it customizable. I like the idea of maintaining different versions for different departments so we can make the change in one place. Do you think it makes sense to do that with the color presets also? We do build a few websites for the same departments. My vote would be to track the footer as as separate issue.

**Comment by stdavis** [comment]
> Do you think it makes sense to do that with the color presets also?

I'm good with the colors also.
